rgbd/include/rgbd/image.h

```diff
@@ -40,29 +41,28 @@ enum DepthStorageType
 
 class Image {
 
-    friend class Server;
-    friend class ServerShmOnly;
-    friend class Client;
-    friend class SharedMemClient;
-    friend class View;
+    friend class ServerRGBD;
```

Comment:
It reduces the copy operations, I will check once more to see if the same performance can be achieved without friend classes.

Reply:
Only `ClientSHM` remains. I think the performance of it would be reduced, see https://github.com/tue-robotics/rgbd/blob/ed5a3dd27aa315019d51b563efc2a8f33f64bf2d/rgbd/src/client_shm.cpp#L95